charts/mirrorbits-parent/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.serviceaccount.enabled }}
+# From https://github.com/jenkinsci/kubernetes-plugin/blob/master/src/main/kubernetes/service-account.yml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "mirrorbits-parent.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+{{- end }}
+
+---
+{{- if .Values.serviceaccount.enabled }}
+# From https://github.com/jenkinsci/kubernetes-plugin/blob/master/src/main/kubernetes/service-account.yml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "mirrorbits-parent.serviceAccountName" . }}-role
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["create","delete","get","list","patch","update","watch"]
+- apiGroups: [""]
+  resources: ["pods/exec"]
+  verbs: ["create","delete","get","list","patch","update","watch"]
+- apiGroups: [""]
+  resources: ["pods/log"]
+  verbs: ["get","list","watch"]
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["watch"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get"]
```

Comment:
Blocker: we only want permissions for an "exec" operation. We don't want ability to read secrets or logs for instance.